fastestimator/trace/trace.py

```diff
@@ -168,10 +168,8 @@ def _print_message(self, header: str, data: Data, log_epoch: bool = False):
         log_message = header
         if log_epoch:
             log_message += "epoch: {}; ".format(self.system.epoch_idx)
-        for key, val in data.read_logs(to_set(self.inputs)).items():
+        for key, val in data.read_logs(self.inputs).items():
```

Comment:
still need to_set to match read_logs type spec

Reply:
it's already a set

Reply:
trace inputs are all converted to_list in super init right?

Reply:
oh right!  changed it